src/main/java/fi/hsl/common/transitdata/proto/PubtransTableProtos.java

```diff
@@ -20,15 +20,15 @@ public static void registerAllExtensions(
 
```

Comment:
Why this file has changed?

Reply:
No idea, maybe generate protos uses some old version of something. Reverting that file.